spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe QueueItem, :type => :model do
+
+  it { is_expected.to belong_to(:user) } 
+  it { is_expected.to belong_to(:video) } 
+
+  describe "#video_title" do
```

Comment:
You can refactor these even more - check [shoulda matchers documentation](https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/independent/delegate_method_matcher.rb) for `delegate_method` matcher.